### PR TITLE
Reorganize files in frontend/src/scenes/insights/

### PR DESCRIPTION
**Pull request by samwinslow**
## Changes

Adds to #4050 option B. This PR contains very simple reorganization of files (note new file Sortable.tsx), name changes, and the addition of a `verbose` prop on `ActionFilter`, triggered at the high level by feature flag, which will enable the layout of Option B.

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious

### CODE REVIEW

**Comment by samwinslow** [comment]
@paolodamico this was the type of thing i mentioned in standup - rename / reorganize before the main logic work gets done

**Comment by paolodamico** [comment]
Perfect, yes! Is this ready for a review? (as it's marked as draft). Also personally I find it easier if you detail the changes a bit more, but that's totally personal preference.

**Comment by samwinslow** [comment]
> Perfect, yes! Is this ready for a review? (as it's marked as draft). Also personally I find it easier if you detail the changes a bit more, but that's totally personal preference.

Apologies, yes it’s ready for review. I should have elaborated that the feature flag is called `4050-query-ui-optB` and that right now this flag is not ready to be enabled.

File: frontend/src/scenes/insights/ActionFilter/ActionFilter.tsx [hunk 1]
```diff
@@ -4,69 +4,25 @@ import { useActions, useValues } from 'kea'
...
     hideMathSelector?: boolean
     hidePropertySelector?: boolean
-    copy: string
+    buttonCopy: string
     disabled?: boolean
     singleFilter?: boolean
     sortable?: boolean
     showLetters?: boolean
     showOr?: boolean
+    verbose?: boolean
```
Comment (paolodamico) [inline frontend/src/scenes/insights/ActionFilter/ActionFilter.tsx:25]: Worth adding clarification of what this attribute is used for

**Review by paolodamico — APPROVED** [review]
lgtm! Btw it was so much easier to review this without any logic changes. Definitely the way to go when refactoring a large part of file structure.

FYI removed the fixed TS error and added some clarification on some props for future reference.

**Pull request merged**